app/uk/gov/hmrc/leakdetection/services/SummaryService.scala

```diff
@@ -89,8 +97,23 @@ class SummaryService @Inject()(ruleService: RuleService,
   }
 
   private def getUnresolvedLeakCount(leaks: Seq[Leak]): Int = leaks.filterNot(_.isExcluded).length
+
   private def getExcludedLeakCount(leaks: Seq[Leak]): Int   = leaks.filter(_.isExcluded).length
 
+  private def getFirstScanned(leaks: Seq[Leak], warnings: Seq[Warning], activeBranches: Seq[ActiveBranch]): Instant =
+    (leaks, warnings, activeBranches) match {
```

Comment:
should this be `(leaks.map(_.timestamp) ++ warnings.map(_.timestamp) ++ active.map(_.updates)).min` (or `reduce(_ min _)`)  - i.e. min date of any?